src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -1282,6 +1282,13 @@ private void runWithRetries(ContextualRunnable runnable, long retryDelay, String
       .onFailedAttempt(failureContext -> {
         Throwable t = failureContext.getLastFailure();
         LOG.error(onFailedAttemptMessage, t);
+        //Logging error list
+        List<BigQueryError> errors = ((BigQueryException) t).getErrors();
+        if (errors.size() > 1) {
```

Comment:
1. Check errors is not null
2. Why are we not logging if we have a single error (errors.size() == 1) ?